test/layout/_query-utils.scss

```diff
@@ -8,72 +14,72 @@
 @include describe('Nudge Query [function]') {
   @include it('Removes 1px from max px values') {
     @include assert-equal(
-      _a_nudge-query(20px, 'max'),
+      utils.nudge-query(20px, 'max'),
       19px
     );
   }
 
   @include it('Removes 0.01 from non-px max values') {
     @include assert-equal(
-      _a_nudge-query(20em, 'max'),
+      utils.nudge-query(20em, 'max'),
       19.99em
     );
 
     @include assert-equal(
-      _a_nudge-query(20ch, 'max'),
+      utils.nudge-query(20ch, 'max'),
       19.99ch
     );
   }
 
   @include it('Ignores non-adjusted values') {
     @include assert-equal(
-      _a_nudge-query(20px, 'min', 'max'),
+      utils.nudge-query(20px, 'min', 'max'),
       20px
     );
 
     @include assert-equal(
-      _a_nudge-query(20em, 'max', 'min'),
+      utils.nudge-query(20em, 'max', 'min'),
       20em
     );
 
     @include assert-equal(
-      _a_nudge-query(20ch, 'min'),
+      utils.nudge-query(20ch, 'min'),
       20ch
     );
 
     @include assert-equal(
-      _a_nudge-query(20ch, 'min', false),
+      utils.nudge-query(20ch, 'min', false),
       20ch
     );
 
     @include assert-equal(
-      _a_nudge-query(20ch, 'max', false),
+      utils.nudge-query(20ch, 'max', false),
       20ch
     );
   }
 
-  $adjust-query-overlap: 'min' !global;
+  utils.$adjust-query-overlap: 'min' !global;
```

Comment:
These "global" variables are giving me trouble.

Reply:
I had to play around with this to figure it out. What I did gets the tests to pass here. I think it was just making sure that the `$adjust-query-overlap` variable was pointing to the right place. 

On line 60  the original value in `/sass/query-utils` is referenced.
`$_old: utils.$adjust-query-overlap;`

It gets reset in the next line. 
`utils.$adjust-query-overlap: 'min';`

And after the tests are run, it gets reset to it's original value. 
`utils.$adjust-query-overlap: $_old;`

@mirisuzanne will be able to explain this better but I think that's what's happening here.

Reply:
The `!global` flag says to 'update the global version of this variable, rather than a nested version of it'. But once you namespace a variable from another module, that's already clear. There is no local version of `utils.$whatever`, just the one in `utils` module. So you don't need (and Sass won't allow) the global flag in that case. 

There are still times you might use it, to change a variable at the root of a module, from inside a mixin, for example. But we will generally be able to remove it in most cases, when there is a namespace involved.